infrastructure/helm/tigerbeetle/templates/configmap.nginx.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Chart.Name }}-nginx-proxy
+data:
+  # property-like keys; each key maps to a simple value
+  nginx-prestart.sh: |
```

Comment:
Dynamically build the nginx config because the DNS server address is only known at runtime. and it is required for the `resolve` directive for the backends.